finagle-netty4/src/test/scala/com/twitter/finagle/netty4/ssl/Netty4SslAddressTest.scala

```diff
@@ -18,10 +18,12 @@ class Netty4SslAddressTest extends FunSuite {
   }
 
   private[this] val hasAddressSessionVerifier = new SslServerSessionVerifier {
-    def apply(address: Address, config: SslServerConfiguration, session: SSLSession): Boolean =
-      address match {
-        case Address.Inet(isa, _) => !isa.isUnresolved
-        case _ => false
+    def apply(address: Address, config: SslServerConfiguration, session: SSLSession): Future[Boolean] =
+      Future.value {
+        address match {
```

Comment:
Can we make this:

    address match {
        case Address.Inet(isa, _) if !isa.isUnresolved => Future.True
        case _ => Future.False
    }